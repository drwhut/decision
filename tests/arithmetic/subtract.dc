Start~#1

< Subtracting 2 positive integers >
Subtract(725, 42)~#2
Print(#1, #2)~#3

< Subtracting a negative integer from a positive integer >
Subtract(15, -5)~#4
Print(#3, #4)~#5

< Subtracting a positive integer from a negative integer >
Subtract(-20, 100)~#6
Print(#5, #6)~#7

< Subtracting 2 negative integers >
Subtract(-5, -7)~#8
Print(#7, #8)~#9

< Subtracting 2 positive floats >
Subtract(0.5762, 0.1399)~#10
Print(#9, #10)~#11

< Subtracting a negative float from a positive float >
Subtract(123.456, -64.32)~#12
Print(#11, #12)~#13

< Subtracting a positive float from a negative float >
Subtract(-7.09, 10.12)~#14
Print(#13, #14)~#15

< Subtracting 2 negative floats >
Subtract(-12000.0, -6014.25)~#16
Print(#15, #16)~#17

< Subtracting a float from an integer >
Subtract(12, 5.3)~#18
Print(#17, #18)~#19

< Subtracting an integer from a float >
Subtract(4.23, 1)~#20
Print(#19, #20)~#21