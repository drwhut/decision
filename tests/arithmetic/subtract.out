683
20
-120
2
0.4363
187.776
-17.21
-5985.75
6.7
3.23
